third_party/move/documentation/coding_guidelines.md

```diff
@@ -23,6 +23,29 @@ $ cargo xclippy --all-targets
 
 In general, we follow the recommendations from [rust-lang-nursery](https://rust-lang.github.io/api-guidelines/) and [The Rust Programming Language](https://doc.rust-lang.org/book/).  The remainder of this guide provides detailed guidelines on specific topics in order to achieve uniformity of the codebase.
 
+## Code review
+
+We require every PR to have approval by at least two reviewers. This is enforced by the CI presubmit.
+
+### Guidelines for Authors of PRs
+
+- Have a meaningful description of the PR
```

Comment:
Link to fixed issues.

When merging the PR as as commit(s), be sure to include
- purpose of the change
- references to fixed issues and design documents

Remember that future code maintainers will use your commit messages to understand why the code was changed and why it is written the way it is.

Reply:
Btw, these were suggested text additions.  Not comments to you.  Sorry for the lack of framing.

Reply:
Done